lib/random.js

```diff
@@ -24,14 +24,14 @@ module.exports = function (base64, crypto, promise) {
         },
 
         /**
-         * Generates a random password fulfilling the size requirements.
+         * Generates a random string fulfilling the size requirements.
          * Does this by generating sufficient random data, converting it
          * to Base64, then doing some string manipulation.
          *
          * @param {number} size
          * @return {Promise.<string>}
          */
-        passwordAsync: function (size) {
+        randomizedAsync: function (size) {
```

Comment:
Rename to something that is more aptly describing what this does or what it makes.

```
randomIdAsync: function (size) {
```